mopro-ffi/Cargo.toml

```diff
@@ -71,6 +73,12 @@ ark-bls12-381 = { version = "0.4.0", optional = true }
 num-traits = { version = "0.2.0", optional = true }
 anyhow = "1.0.86"
 
+# Nova Scotia dependencies
+nova-scotia = "0.5.0"
+nova-snark = "0.23.0"
+serde_json = "1.0.85"
+pasta_curves = "0.5"
```

Comment:
these dependencies should be set `optional`
e.g.
```toml
nova-scotia = { version="0.5.0", optional = true}
```

Reply:
Testing.
I move those dependencies after [dev-dependencies].

Reply:
you still need to set it `optional` since it is in `nova_scotia` feature
the error is
```sh
`cargo metadata` exited with an error: error: failed to load manifest for workspace member `/home/runner/work/mopro/mopro/mopro-ffi`
referenced by workspace at `/home/runner/work/mopro/mopro/Cargo.toml`

Caused by:
  failed to parse manifest at `/home/runner/work/mopro/mopro/mopro-ffi/Cargo.toml`

Caused by:
  feature `nova_scotia` includes `nova-scotia`, but `nova-scotia` is not an optional dependency
  A non-optional dependency of the same name is defined; consider adding `optional = true` to its definition.

...
```

I suppose all the nova-scotia circuits will use the features like `nova-scotia`?
(and even all the dependencies??)
it won't be effected by the circuit's difference
e.g. I use `fibonacci` circuit, and I use `poseidonHash` circuit, I will use the same `nova-scotia` (but different `r1cs` and `wasm`)